deploy/97_fixtureDeployments.ts

```diff
@@ -54,7 +54,10 @@ const func = async function (env: HardhatRuntimeEnvironment) {
   );
 
   // Configure contract roles
-  console.log("Configuring strategy contract role");
+  console.log("Configuring contract roles");
+  const SPONSOR_ROLE = utils.keccak256(utils.toUtf8Bytes("SPONSOR_ROLE"));
+  await vault.connect(owner).grantRole(SPONSOR_ROLE, treasury.address);
```

Comment:
I'm confused, why do you need the treasur to sponsor?

Reply:
https://github.com/lindy-labs/sc_solidity-contracts/commit/d6b2fbf02f6f08debcb58b1378ffe9492236b742

was introduced yesterday

Reply:
I know, but our treasury is a sponsor? I wasn't aware of that in our deployment scripts

Reply:
ah! sorry misunderstood you

the 98_fixtures.ts file configures the treasury as a sponsor

https://github.com/lindy-labs/sc_solidity-contracts/blob/main/deploy/98_fixtures.ts#L32-L35